Multisig/App/SceneDelegate.swift

```diff
@@ -62,12 +79,25 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
         } else {
             onAppUpdateCompletion()
         }
+
+        // Check if we have copied WalletConnect url in the Pasteboard and handle it
+        if let potentialWCUrl = Pasteboard.string, potentialWCUrl.hasPrefix("wc:") {
+            do {
+                App.shared.snackbar.show(message: "Creating WalletConnect session. This might take some time.")
+                try WalletConnectServerController.shared.connect(url: potentialWCUrl)
+                // if setting nil it will crash
+                Pasteboard.string = ""
+            } catch {
+                App.shared.snackbar.show(message: "Failed to create a WalletConnect session.")
```

Comment:
potential conflict with the current window logic: update app screen and passcode screens

Reply:
probably better to do it in the main screen

Reply:
added to further improvements tasks